src/utils/mutex.h

```diff
@@ -127,6 +127,13 @@ class CAPABILITY("mutex") SharedMutex {
 static inline void SpinloopPause() {
 #if defined(__x86_64__)
```

Comment:
This needs to be `defined(__x86_64__) || defined(_M_X64)` as msvc doesn't support `__asm` for x86_64 but does support `_mm_pause()`. This will also break arm cross builds with msvc (again no `__asm`), but I'll deal with it when I update #1801.